.github/workflows/build.yml

```diff
@@ -105,8 +106,23 @@ jobs:
           img="$(jq <<<"$json" -r '.build.img')"
           if crane digest "$img"; then
             echo >&2 "error: '$img' already exists! cowardly refusing to overwrite it"
-            exit 1
+            echo 'cancel=exists' >> "$GITHUB_OUTPUT"
+          else
+            echo 'cancel=' >> "$GITHUB_OUTPUT"
```

Comment:
I don't know if this is actually necessary, but it felt better to be fully explicit.